backend/src/core/models/job-queue.ts

```diff
@@ -34,4 +34,11 @@ export class JobQueue extends Model<JobQueue> {
     allowNull: false,
   })
   status!: JobStatus
+
```

Comment:
think the current column comment added in the migration should be here (easier to reference compared to the db)

Reply:
fixed w next push